.storybook/preview.js

```diff
@@ -0,0 +1,24 @@
+import * as NextImage from 'next/image'
+import { WalletProvider } from '../components/WalletProvider'
+import { theme } from '../theme'
+
+// de-optimize Image component
+const OriginalNextImage = NextImage.default
+Object.defineProperty(NextImage, 'default', {
+  configurable: true,
+  value: (props) => <OriginalNextImage {...props} unoptimized />,
+})
+
+export const parameters = {
```

Comment:
[`parameters`](https://storybook.js.org/docs/react/writing-stories/parameters) define global configuration that can be accessed by Storybook addons or individual stories.